sdrbase/CMakeLists.txt

```diff
@@ -381,6 +380,7 @@ target_link_libraries(sdrbase
     ${sdrbase_FFTW3F_LIB}
     ${sdrbase_SERIALDV_LIB}
     ${sdrbase_LIMERFE_LIB}
+    Boost::headers
```

Comment:
Not working in many linux distributions that do not have the proper version of cmake